gradle.properties

```diff
@@ -1,4 +1,5 @@
 android.defaults.buildfeatures.buildconfig=true
 android.nonFinalResIds=false
 android.nonTransitiveRClass=false
-android.useAndroidX=true
\ No newline at end of file
+android.useAndroidX=true
+org.gradle.jvmargs=-Xmx2g
```

Comment:
It seems to me that this should be a local setting and not part of this PR

Reply:
I got OOM's without increasing the heap. Gradle properties in a project are a tricky thing though. I will try building again, maybe just add it in the docs if the heap needs to be increased.

Reply:
If it is really required to build, then it should be included in the PR

Reply:
Seems to build fine now without specifying jvm args.